Cataclysm/Warrior.lua

```diff
@@ -610,42 +636,43 @@ spec:RegisterAuras( {
 
 -- Glyphs
 spec:RegisterGlyphs( {
+    [58095] = "Battle",
```

Comment:
These need to be in simc token format (lower case, spaces replaced with underscores, no special characters).

I.e., `berserker_rage`, `bladestorm`, `bloodthirst`, `bloody_healing`...